src/dotnet/Audio.UI.Blazor/Components/AudioRecorder/AudioRecorderCommands.cs

```diff
@@ -3,7 +3,7 @@ namespace ActualChat.Audio.UI.Blazor.Components;
 public interface IAudioRecorderCommand
 { }
 
-public sealed record StartAudioRecorderCommand(ChatId ChatId, CancellationTokenSource RecordingCancellation) : IAudioRecorderCommand;
+public sealed record StartAudioRecorderCommand(ChatId ChatId) : IAudioRecorderCommand;
```

Comment:
Why CTS is dropped here?

Reply:
(just asking)

Reply:
it was used to cancel in command to cancel StartRecording command itself.
Now MessageProcessor conducts cancellation and we await MessageProcess to complete. MessageProcess has it's cancellation timeout, provides cancellationToken to command handler, etc. 
As for me it looks much more predictable and straight forward.